src/main/mjml/dummy/test-email-change-notification.html

```
@@ -0,0 +1,4 @@
+To generate proper html code use the template in src/main/mjml with an mjml parser (e.g. use the IntelliJ plugin for MJML or follow the instructions on https://documentation.mjml.io/#usage
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)